### PR TITLE
Added tests for project page

### PR DESCRIPTION
**Pull request by eanker**
Added some test for the project page and extracted a variable from the list navigation tests.

### CODE REVIEW

**Comment by eanker** [comment]
CI is failing, because of the to-date-string tests.

File: test/projects/projects-page.html [hunk 1]
```diff
@@ -33,8 +33,30 @@
         projectPage.showProjectsList = true;
         MockInteractions.tap(Polymer.dom(projectPage.root).querySelector('iron-pages'));
         assert.isFalse(projectPage.showProjectsList);
+      });
+
+      test('Selecting project by title', function() {
```
Comment (txsmith) [inline test/projects/projects-page.html]: The test name do not correspond to the usual format:
`sets activeProject when project title is set`

File: test/projects/projects-page.html [hunk 2]
```diff
@@ -33,8 +33,30 @@
...
+        projectPage.projectSelected = false;
+        assert.equal('no-selected', Polymer.dom(projectPage.root).querySelector('iron-pages').selected);
+      });
+
+      test('Show overview page when a project is selected', function() {
+        projectPage.projectSelected = true;
+        assert.equal('overview', Polymer.dom(projectPage.root).querySelector('iron-pages').selected);
+      });
 
+      test('Show pull request page when a pull request is selected', function() {
```
Comment (txsmith) [inline test/projects/projects-page.html]: Maybe also change `Show` to `shows`?

**Comment by txsmith** [comment]
Looks good!

**Pull request merged**